lib/broccoli/bundler.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const concatFiles = require('./concat-files');
+const processModulesOnly = require('./babel-process-modules-only');
+
+class Bundler {
+  /*
+    Responsibility of this class is to intelligently produce
+    the final output given a list of trees that were passed in.
+
+    @class Bundler
+    @constructor
+    @param {Object} Configuration options
+   */
+  constructor(options) {
+    this.name = options.name;
+    this.sourcemaps = options.sourcemaps;
+    this.appOutputPath = options.appOutputPath;
+    this.isBabelAvailable = options.isBabelAvailable;
+    this.appTreeAnnotation = options.appTreeAnnotation;
```

Comment:
it seems odd to have the annotation here, but expect the returned tree to come from the `bundleAppJs`